superset/viz.py

```diff
@@ -2107,6 +2108,10 @@ def tupleify(s):
             df[key] = list(zip(latlong.apply(lambda x: x[0]),
                                latlong.apply(lambda x: x[1])))
             del df[spatial.get('geohashCol')]
+
+        if df[key] is None:
```

Comment:
(optional)`df.get(key) is None:` would protect against KeyError (not sure if they're possible in this context)